cldfbench_doreco.py

```diff
@@ -15,21 +20,46 @@ def cldf_specs(self):  # A dataset must declare all CLDF sets it creates.
                 module='StructureDataset'
                 )
 
+    def cmd_download(self, args):
+        self.raw_dir.download(
+            'https://sharedocs.huma-num.fr/wl/?id=s947TcfRfDZR643QURdyncdku4EmeKyb&fmode=download',
+            'languages.csv')
+        self.raw_dir.download(
+            'https://sharedocs.huma-num.fr/wl/?id=xqkUR3WoFOKB8cRb0MdHYJxIDEXEXlVG&fmode=download',
+            'sources.bib')
+
+        for row in self.raw_dir.read_csv('languages.csv', dicts=True):
+            if 'ND' not in row['Annotation license']:
+                print(row['Glottocode'], row['DOI'])
+                cid = urllib.parse.quote(row['DOI'], safe='')
+                for f in requests.get("{}datas/{}?metadata-format=dc".format(NAKALA_API, cid)).json()['files']:
```

Comment:
Retrieve metadata about files of a corpus from nakala, using the [API](https://api.nakala.fr/doc).